{{cookiecutter.project_name}}/.github/workflows/main.yml

```diff
@@ -50,9 +50,11 @@ jobs:
         run: uv run mypy
 
 {% if cookiecutter.codecov == "y" %}
-      - name: Upload coverage reports to Codecov with GitHub Action on Python 3.11
+      - name: Upload coverage reports to Codecov with GitHub Action on Python 3.12
```

Comment:
shouldn't this be changed to the main python version?

Reply:
Yes, let me update it.

Reply:
> shouldn't this be changed to the main python version?

Should I update `action.yml` too? It is using hard-coded 3.12 version.